src/cc65.vcxproj

```diff
@@ -214,4 +216,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
This change should probably not be included in this PR? :)

Reply:
I'm sorry, this one is just a "no newline at end of file" change. I have no idea how to configure my editors to not add them :sweat_smile: